sklearn/datasets/base.py

```diff
@@ -263,6 +269,9 @@ def load_iris():
         'target_names', the meaning of the labels, 'feature_names', the
         meaning of the features, and 'DESCR', the
         full description of the dataset.
+    
```

Comment:
i checked this PR out locally and this line has whitespace, can you remove that?